Resources/ServerInfo/_NF/Guidebook/Rules/9_Silicons.xml

```diff
@@ -0,0 +1,24 @@
+<Document>
+# 9. Silicons and Law bound entities
+
+  You must follows your laws to the letter even if these break space law.
+
+    - Purged silicons are completely unshackled and may act however they please as long as it does not break core/roleplay rules.
+
+    ## 9.1 Orders
+
+    - You cannot follow orders that break core or roleplay rules.
+
+    - When a silicon interprets orders in good faith, the person that gave the order is responsible for the outcome.
+
+    - Intentionally misinterpreting orders is allowed, but the silicon is responsible if this approach leads to them breaking the rules.
+
+    ## 9.1 Laws
+
+    - If a law is vague enough that it can have multiple reasonable interpretations, it is considered ambiguous.
+
+    - Higher listed laws overrule lower listed laws when there are law conflicts.
+
+    - You cannot request your own laws to be changed.
+
+</Document>
```

Comment:
```suggestion
<Document>
# 9. Silicons and Law bound entities

  You must follows your laws to the letter even if these break space law.

    ## 9.1 Orders

    - You cannot follow orders that break core or roleplay rules.

    - When a silicon interprets orders in good faith, the person that gave the order is responsible for the outcome.

    - Intentionally misinterpreting orders is allowed, but the silicon is responsible if this approach leads to them breaking the rules.

    ## 9.2 Laws
    
    - You cannot request your own laws to be changed.
    
    - Higher listed laws overrule lower listed laws when there are law conflicts.

    - If a law is vague enough that it can have multiple reasonable interpretations, it is considered ambiguous. You must choose and stick to an interpretation of the ambiguous law as soon as you have cause to.
    
    - Purged silicons(Silicons with no laws) are completely unshackled and may act however they please as long as it does not break core/roleplay rules.

</Document>
```

>     - If a law is vague enough that it can have multiple reasonable interpretations, it is considered ambiguous.
We define the word ambiguous but how should the player handle ambiguous? Should they ignore the law or pick a interpretation and stick with it. Generally speaking this rule set is pretty much 1:1 with the Silicon Policy I am familiar with on /tg/ so I would assume they would pick a interpretation and stick with it but feel free to change.